### PR TITLE
Refactor: Compile service & add rollup for Storage

### PR DESCRIPTION
**Pull request by SangTran-127**
## Type

- [x] Feature
- [x] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/common/src/types/group.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export type TGroup = {
   groupName: string;
   groupDescription: string;
   // It's userName
-  createdBy: string;
+  createBy: string;
```
Comment (chiro-hiro) [inline packages/common/src/types/group.ts:11]: lol `createBy`

Comment (SangTran-127) [reply packages/common/src/types/group.ts]: Fixed

File: packages/serverless/src/domain/use-case/transaction.ts [hunk 2]
```diff
@@ -143,13 +143,13 @@ export class Transaction {
...
 
-    await redisQueue.enqueue(
-      JSON.stringify({
-        id: insertResult.insertedId.toString(),
-        payerAddress: payer?.publicKey,
-      })
-    );
+    await transactionQueue.add('transaction', {
+      transactionObjectId: insertResult.insertedId,
+      payerAddress: payer?.publicKey,
```
Comment (chiro-hiro) [inline packages/serverless/src/domain/use-case/transaction.ts:151]: What if payer is null or undefined?

Comment (SangTran-127) [reply packages/serverless/src/domain/use-case/transaction.ts:151]: Solved

**Pull request merged**